cmd/goProbe/config/config.go

```diff
@@ -63,6 +63,10 @@ type CaptureConfig struct {
 	Promisc bool `json:"promisc" yaml:"promisc"`
 	// used by the ring buffer in capture
 	RingBuffer *RingBufferConfig `json:"ring_buffer" yaml:"ring_buffer"`
+
+	// LocalBufferSizeLimit denotes the maximum size of the local buffer
+	// used to continue capturing while the capture is (b)locked
+	LocalBufferSizeLimit int `json:"local_buffer_size_limit" yaml:"local_buffer_size_limit"`
```

Comment:
This needs to be included in the `Equals` method, otherwise the interface will not be reloaded upon changes:
```
// Equals compares c to cfg and returns true if all fields are identical
func (c CaptureConfig) Equals(cfg CaptureConfig) bool {
	return c.Promisc == cfg.Promisc && c.RingBuffer.Equals(cfg.RingBuffer) && c.LocalBufferSizeLimit == cfg.LocalBufferSizeLimit
}

```

Reply:
Ahjo, true.